containers/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+load("@rules_oci//oci:defs.bzl", "oci_image", "oci_load")
+load("@tar.bzl", "tar")
+
+package(default_visibility = ["//visibility:public"])
+
+# Packaging the release binary into a tar archive, which is needed by oci_image rule
+tar(
+    name = "tar",
+    srcs = ["//src/apps:release"],
+)
```

Comment:
https://github.com/bazel-contrib/tar.bzl/blob/main/examples/migrate-rules_pkg/BUILD#L36-L44

Should work to adjust the path so that the executable would be some moire suitable path e.g. `/opt/`, but using 

```
mtree_mutate(
    name = "move_to_opt_data",
    mtree = ":mtree",
    owner = "1000",
    package_dir = "/opt/data",
)
```

fails with bazel not passing the mtree data properly as an argument. This needs to be looked into.